packages/dds/merge-tree/src/partialLengths.ts

```diff
@@ -95,6 +95,43 @@ export interface PartialSequenceLength {
     overlapRemoveClients?: RedBlackTree<number, IOverlapClient>;
 }
 
+interface UnsequencedPartialLengthInfo {
+    /**
+     * Contains entries for all local operations.
+     * The "seq" field of each entry actually corresponds to the delta at that localSeq on the local client.
```

Comment:
curious how the reviewer feels about this. It enables some nice code re-use (avoiding either duplication or some added indirection to existing helper functions in this file), but the semantics are a bit more confusing.

Reply:
i think i now understand this based on my comments below. i like the reuse, but it does make the code confusing, and possibly error prone. i wonder if additional refactoring would help. for instance, we could potentially split the methods up so that methods that use partialLengths are separate from those that choose local vs remote. the goal being that with the methods that use partialLengths, there is no ability to look at the wrong sequence space. not sure how feasible that is. I think some inline comment, and maybe some variable could be just as effective, calling things something like `localOrRemoteSeq` rather than just `seq`, as thoughout the code `seq` is generally assumed to be a remote seq.